HoI4Mod/Rome_Strikes_Back/common/decisions/lar_agent_recruitment_decisions.txt

```diff
@@ -696,7 +696,7 @@ lar_local_recruitment = {
 			FROM = {
 				is_on_continent = asia
 				NOT = {
-					OWNER = { has_country_flag = conv_south_asia }

```

Comment:
The scope on these was causing issues. As it's checking a country flag on a state, so the owner scope change is needed.

Reply:
Ah, shoot. I got mixed up and thought this was a converter change that had been made in the meanwhile.